packages/bygger/src/App.js

```diff
@@ -31,6 +32,11 @@ function App({ projectURL, serverURL, pusher }) {
     ? () => <AuthenticatedApp serverURL={serverURL} formio={formio} />
     : () => <UnauthenticatedApp projectURL={projectURL} />;
 
+  const root = document.getElementById("root", {
+    disable: process.env.NODE_ENV === "production",
```

Comment:
Dobbeltsjekk denne. disable? er NODE_ENV satt? burde du sjekke på appConfig development is true?